recipes/newrelic/infrastructure/kubernetes.yml

```diff
@@ -0,0 +1,353 @@
+# Visit our schema definition for additional information on this file format.
+# https://github.com/newrelic/open-install-library/blob/main/docs/recipe-spec/recipe-spec.md#schema-definition
+
+name: kubernetes-open-source-integration
+displayName: Kubernetes Open Source Integration
+description: New Relic install recipe for the Kubernetes Open Source integration
+repository: https://github.com/newrelic/nri-kubernetes
+
+installTargets:
+  - type: host
+    os: linux
+  - type: host
+    os: darwin
+
+keywords:
+  - Infrastructure
+  - Agent
+  - Kubernetes
+  - Pixie
+  - K8s
+  - OpenShift
+  - EKS
+  - GKE
+  - AKS
+
+processMatch: []
+
+successLinkConfig:
+  type: EXPLORER
+
+validationNrql: "SELECT count(*) FROM K8sClusterSample SINCE 10 minutes AGO"
```

Comment:
That's a good point. I think having a local validation approach would be better, however I'm not sure how that could be implemented. For the first release is it better to omit the validationNrql?

Reply:
@fryckbos I would say yes, let s remove the validationNrql.
Can you add a recipe task to do a get pods until one of our newrelic pods gets active? You'll want to have a timeout, a few minutes should do.

Reply:
In case it's helpful, the guided install UI is checking for a cluster with the given name, and then it checks for visualization nodes. (Not sure if we can even query that service from the CLI?) Without that second check the cluster explorer generally isn't ready. If the UX is still the same as it was when this was implemented, we might want to make some improvements if the plan is to send them to the cluster explorer shortly after the NR pods are active.

Reply:
I agree, we need a follow work item to wait and check for the kubernetes data to make its way to NR so our K8 explorer shows something.
@fryckbos do you have another ticket, and or branch?